crates/runtime/src/externref.rs

```diff
@@ -649,70 +679,46 @@ impl VMExternRefActivationsTable {
 
     /// Sweep the bump allocation table after we've discovered our precise stack
     /// roots.
-    fn sweep(&self, precise_stack_roots: &mut HashSet<VMExternRefWithTraits>) {
-        // Swap out the over-approximated set so we can distinguish between the
```

Comment:
Very nice!!!

Reply:
Wait -- is this actually true? What about `Drop` impls that call back into Wasm? I guess they *can't* since they don't have a `StoreContextMut`. Nice.